pwiz_tools/BiblioSpec/src/PepXMLreader.cpp

```diff
@@ -419,7 +426,8 @@ void PepXMLreader::startElement(const XML_Char* name, const XML_Char** attr)
            (analysisType_ == PROTEOME_DISCOVERER_ANALYSIS && scoreType_ == MASCOT_IONS_SCORE && score_name == "exp-value") ||
            (analysisType_ == MORPHEUS_ANALYSIS && score_name == "psm q-value") ||
            (analysisType_ == MSGF_ANALYSIS && score_name == "qvalue") ||
-           (analysisType_ == CRUX_ANALYSIS && score_name == "percolator_qvalue")) {
+           (analysisType_ == CRUX_ANALYSIS && score_name == "percolator_qvalue")
+           (analysisType_ == PERCOLATOR_ANALYSIS)) {

```

Comment:
Looks like a missing `||` operator? How did this work?

Reply:
And this is why I shouldnt push changes on a friday afternoon. Fixed.